### PR TITLE
Fix/master remove compound

### PR DESCRIPTION
**Pull request by jjettenn**
#### Checklist
- [x] Unit tests
- [ ] Updated testplan

### CODE REVIEW

**Comment by tommydeboer** [comment]
There are 9 failing tests

**Comment by tommydeboer** [comment]
test this please

**Comment by jjettenn** [comment]
I ran the clean install after the last changes and it works on my machine. Strange!

**Comment by jjettenn** [comment]
test this please

**Comment by jjettenn** [comment]
test this please

**Comment by jjettenn** [comment]
test this please

**Comment by jjettenn** [comment]
test this please

**Comment by jjettenn** [comment]
test this please

File: molgenis-data/src/main/java/org/molgenis/data/meta/AttributeRepositoryDecorator.java [hunk 1]
```diff
@@ -271,6 +272,17 @@ public void delete(Attribute attr)
 	{
 		validateDeleteAllowed(attr);
 
+		// If compound attribute is deleted then change the parent of children to null
+		// This will change the children attributes into regular attributes.
+		if(AttributeType.COMPOUND.equals(attr.getDataType()))
+		{
+			attr.getChildren().forEach(e -> {
```
Comment (fdlk) [inline molgenis-data/src/main/java/org/molgenis/data/meta/AttributeRepositoryDecorator.java]: e -> rename to childAttribute ?

**Comment by dennishendriksen** [comment]
@fdlk Are you reviewing this pull request? You added comments without assigning yourself to this PR.

**Comment by fdlk** [comment]
@dennishendriksen I unassigned myself cause the story is not in the sprint.

**Comment by fdlk** [comment]
@dennishendriksen Yes, I am.

File: molgenis-data/src/main/java/org/molgenis/data/meta/AttributeRepositoryDecorator.java (hunk 1 above)
Comment (fdlk) [inline molgenis-data/src/main/java/org/molgenis/data/meta/AttributeRepositoryDecorator.java]: Or leave it like this :)

**Pull request merged**